SeriesMaster-Frontend/App/Screens/DiscoverScreen.js

```diff
@@ -31,13 +41,24 @@ const DiscoverScreen = ({ navigation }) => {
   const [query, setQuery] = useState("");
   const [tvShows, setTvShows] = useState([]);
   const debouncedQuery = useDebounce(query, 500);
+  const responseListener = useRef();
 
   useEffect(() => {
+    registerForPushNotifications();
+
+    responseListener.current = Notifications.addNotificationResponseReceivedListener(
```

Comment:
You don't need `useRef` here. 
It should be enough to define the variable here:

```suggestion
    const responseListener = Notifications.addNotificationResponseReceivedListener(
```